js/ui/themes.js

```diff
@@ -334,28 +320,6 @@ function isGeneric(themeName) {
     return isTheme("generic", themeName);
 }
 
-function isWin8(themeName) {
-    return isTheme("win8", themeName);
-}
-
-function isWin10(themeName) {
-    return isTheme("win10", themeName);
-}
-
-function checkThemeDeprecation() {
```

Comment:
@BingoRUS @jauseg 
maybe we need to save this function to use it for the ios7 theme deprecation?

Reply:
I have reverted this functional with the https://github.com/DevExpress/DevExtreme/pull/7497/commits/9944adcab4bb71366bda88a190728866e3a0c1b1 commit